services/buildbot/master/twisted_factories.py

```diff
@@ -523,7 +523,7 @@ class TwistedVirtualenvReactorsBuildFactory(TwistedBaseFactory):
     treeStableTimer = 5*60
 
     def __init__(self, source, RemovePYCs=RemovePYCs, python="python",
-                 trial="./bin/trial", virtualenv_module="virtualenv",
+                 trial="-m twisted.trial", virtualenv_module="virtualenv",
```

Comment:
in this context, do we still need the trial argument at all?

I am assuming that all tests will now run with `-m twisted.trial`